ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -281,20 +281,28 @@
   CompletableFuture<ConnectorDescription> describeConnector(String connectorName);
 
   /**
-   * Define a session variable which can be referenced in sql commands by wrapping it with ${}
+   * Define a session variable which can be referenced in sql commands by wrapping the variable name
+   * with {@code ${}}.
```

Comment:
A quick google search suggests that this won't work -- the inner curly brace causes the code format to terminate prematurely, and needs to be escaped.

Reply:
I think this might have been fixed recently - it showed up properly when I generated the javadocs locally:
<img width="809" alt="Screen Shot 2021-04-05 at 2 10 11 AM" src="https://user-images.githubusercontent.com/11086267/113594292-5b35b900-95ec-11eb-98d9-5fc0cdb15c87.png">

Reply:
Oh okay, good to know. Thanks for checking!